src/frequenz/sdk/config/_config_managing.py

```diff
@@ -106,9 +106,42 @@ def __init__(
             for config_path in config_paths
         ]
         self._output: Sender[abc.Mapping[str, Any]] = output
-        self._event_types: abc.Set[EventType] = event_types
-        self._force_polling: bool = force_polling
-        self._polling_interval: timedelta = polling_interval
+
+        # FileWatcher instance should be create here instead of in _run().
+        # Otherwise, if the actor restarts, we will lost all the events that happened
+        # while the actor was down.
```

Comment:
Case before my updates:
file was deleted -> signal DELETE -> ConfigManager crashed -> it took 2 sec to restart -AND new file was created but we didn't detect it -> new File Watcher was created

Reply:
Ok I gave it a second thought and this was too over complicated and also not working solution.
What I wanted was to watch for files all the time - even if actor crashes. Because during actor crash file can be replaced and we will skip this change. But I just realized we send new config at the beginning of `_run` method. So new config will be send on every actor re-run. So this change is not needed.

I removed this commit but I added new one. But found new bug related with above and fixed it in last(new) commit.